src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arrDate = date.split(fromFormat[3]);
+  const slash = toFormat[3];
```

Comment:
Similarly, `toFormat[3]` is used as the separator for joining the new date format. Ensure that this index is correct for all possible formats or handle separators separately.